testdata/project-v2-multigroup/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -1,5 +1,5 @@
-# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
-# it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+# This patch inject a sidecar container which is a HTTP proxy for the 
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
```

Comment:
How can this pass the `lll` test if it the second line is longer after the change than the first line before the change?

Reply:
This line is in the correct size. 
The lint issue was in the line where the const is defined.

Reply:
This sounds like a bug in lll linter...
I guess we can keep what we have now in the PR for now.

Reply:
It is not. The GitHub diff is making harder you folks get what was a change here.
The change/fix is regarding the above line where the const is defined.